src/mozillavpn.cpp

```diff
@@ -2281,6 +2281,8 @@ void MozillaVPN::registerExternalOperations() {
         new TaskControllerAction(TaskControllerAction::eDeactivate));
   });
 
+  eoh->registerExternalOperation(ExternalOpHandler::OpCloseEvent, []() {});
```

Comment:
Should this be registered on all platforms, since some platforms don't support this?

Reply:
Within `bool Navigator::eventHandled()`, we check for eligible platforms before running the relevant code - it's wrapped in `#if defined(MZ_ANDROID) || defined(MZ_WASM)`. Do you think that suffices, or should we similarly wrap this here?

Reply:
Also as a sidenote - this line will be removed when the PR Lesley mentioned gets merged in - so this is a temporary thing either way.

Reply:
Sounds good.